css-view-transitions-1/Overview.bs

```diff
@@ -1580,7 +1583,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	<div algorithm>
 		The <dfn>capture rendering characteristics</dfn> are as follows:
 
-		* The origin of |element|'s [=ink overflow rectangle=] is anchored to canvas origin.
+		* The origin of |element|'s [=border box=] is anchored to canvas origin.
```

Comment:
I think the existing spec text in capture rendering characteristics doesn't need to change. It's meant to theoretically explain how the image would be generated if no clipping was required and the painted output should cover the element's ink overflow rectangle. So we'd allocate a canvas equal to ink overflow bounds with the origin of the canvas being the ink overflow rectangle's origin.

What you have here should go into step 8.6.2. We can delete the "computing the interest rectangle" algorithm altogether and also steps 2 and 3 in 8.6.2. Instead add what you have here:

* The UA may clip the rendered output from step 8.6.2.1 but it should include what's in the visible viewport and the rasterization quality should be optimal for the scale at which this element renders in the DOM (this is the point that we can discuss tomorrow).
* The returned image has a natural size equal to the element's border box and its origin maps to the border box's origin in the rendered output in step 1. The fact that the underlying pixel data may be smaller or larger than this is a UA detail. The Note below covers this well.

Reply:
Thanks, applied this, please look if this is what you meant.